client/components/site-selector/add-site.tsx

```diff
@@ -19,15 +19,13 @@ const SiteSelectorAddSite: FunctionComponent = () => {
 	}, [ dispatch ] );
 
 	return (
-		<span className="site-selector__add-new-site">
-			<Button
-				borderless
-				href={ `${ onboardingUrl() }?ref=calypso-selector` }
-				onClick={ recordAddNewSite }
-			>
-				<Gridicon icon="add-outline" /> { translate( 'Add new site' ) }
-			</Button>
-		</span>
+		<Button
+			className="site-selector__add-new-site"
```

Comment:
@danielbachhuber I didn't think we'd need to define all the custom colours for this button, because from the mockups I got the impression it was just supposed to be a "primary" button.

```suggestion
			primary
```

![Sep-07-2022 09-45-52](https://user-images.githubusercontent.com/1500769/188746948-08136924-e1ef-4876-a9eb-42a5018f994c.gif)


There are some themes where the hover colour for primary buttons are a little odd.

![Sep-07-2022 09-45-48](https://user-images.githubusercontent.com/1500769/188746714-5812d0ce-4a61-46f9-a201-ab7c02e38241.gif)

But the users will be used to it, since it's the same primary button hover used everywhere.

Reply:
> There are some themes where the hover colour for primary buttons are a little odd.

@p-jackson I think I like my styles better...

Reply:
Well they are better ❤️
Fair enough, I'm fine with the custom colours. The inverted colour scheme of the sidebar justifies the deviation from the norm I think.